numpy/core/src/multiarray/datetime.c

```diff
@@ -2256,15 +2256,11 @@ NpyDatetime_ConvertPyDateTimeToDatetimeStruct(
         else {
             PyObject *offset;
             int seconds_offset, minutes_offset;
-
-            /* 2016-01-14, 1.11 */
-            PyErr_Clear();
-            if (DEPRECATE(
-                    "parsing timezone aware datetimes is deprecated; "
-                    "this will raise an error in the future") < 0) {
-                Py_DECREF(tmp);
-                return -1;
-            }
+            if (PyErr_WarnEx(PyExc_UserWarning,
+                "no explicit representation of timezones available for np.datetime64",
```

Comment:
This seems not very clear to me?  Maybe also mention that parsing to naive datetime but handling timezone as if we were UTC?

(If that is correct)